### ISSUE TITLE
Focus on page load

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#28]
Make sure focus defaults to top of page on page load (unless specific reason not to)
Actions list -> Action focus fails
Indicators list -> Indicator focus fails

WCAG 2.1  1.3.2 A
